releaser/util.go

```diff
@@ -49,7 +49,7 @@ func (r Releaser) githubRelease(
 		return err
 	}
 
-	commit, err := dag.Version().Git().Commit(target).Commit(ctx)
+	commit, err := dag.Version().Git().Commit(src).Commit(ctx)
```

Comment:
Are you sure that we want `src` here @jedevc ?

If we do, then we should remove the `target` argument because it is unused and `engine lint` fails.

Reply:
Fixed, it should have been used later. Clarified further to rename these to `src` and `dest`.